src/nvim/globals.h

```diff
@@ -116,12 +116,13 @@
 #define DFLT_COLS       80              /* default value for 'columns' */
 #define DFLT_ROWS       24              /* default value for 'lines' */
 
-EXTERN long Rows                        /* nr of rows in the screen */
 #ifdef DO_INIT
-  = DFLT_ROWS
+  EXTERN long Rows = DFLT_ROWS;          // nr of rows in the screen
+#else
+  EXTERN long Rows;
 #endif
```

Comment:
Do we need macro `DO_INIT` here? Is this whole block not equivalent to `EXTERN long Rows INIT(= DFLT_ROWS);`?

Looking at the other uses of macro `DO_INIT`, we could remove it completly from this file, if we change macro `INIT` into a variadic macro:
```diff
diff --git a/src/nvim/globals.h b/src/nvim/globals.h
index 52eebeb..197a609 100644
--- a/src/nvim/globals.h
+++ b/src/nvim/globals.h
@@ -7,11 +7,10 @@
 // actually defined and initialized.
 #ifndef EXTERN
 # define EXTERN extern
-# define INIT(x)
+# define INIT(...)
 #else
 # ifndef INIT
-#  define INIT(x) x
-#  define DO_INIT
+#  define INIT(...) __VA_ARGS__
 # endif
 #endif

```
So something like this
```c
EXTERN typebuf_T typebuf                /* typeahead buffer */
#ifdef DO_INIT
  = {NULL, NULL, 0, 0, 0, 0, 0, 0, 0}
#endif
;
```
could be changed into
```c
EXTERN typebuf_T typebuf  INIT(= {NULL, NULL, 0, 0, 0, 0, 0, 0, 0});              /* typeahead buffer */
```